be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -426,6 +431,20 @@ Status FileColumnIterator::_read_page(const OrdinalPageIndexIterator& iter, Pars
     RETURN_IF_ERROR(_reader->encoding_info()->create_page_decoder(data, options, &page->data_decoder));
     RETURN_IF_ERROR(page->data_decoder->init());
 
+    if (_reader->encoding_info()->encoding() == DICT_ENCODING) {
```

Comment:
Better to add comment to explain why we create dict decoder lazily.

Reply:
👌